app/models/section.rb

```diff
@@ -159,14 +160,18 @@ def eql?(other)
     id.eql?(other.id)
   end
 
-protected
-
-  attr_reader :slug_generator, :edition_factory
-
   def never_published?
     !published?
   end
 
+  def change_note_required?
+    !(never_published? || minor_update?)
```

Comment:
@chrisroos: As I alluded to in the PR description I did consider this, but decided that `never_published?` seems to be communicating something a bit different to `!published?` and that method is used elsewhere. I guess I could introduce `ever_published?`. I'll do that now.